### ISSUE TITLE
npm install blocked

### ISSUE THREAD

**Issue by DeviaVir** [butterproject/butter#6]
Looks like this project is dependent off of a resource that has a tendency to go down (https://git.popcorntime.io/mirrors/chromecast-js.git), you might want to host that elsewhere (GH?).

Log:
```
$ npm install
npm ERR! git clone --template=/Users/Chase/.npm/_git-remotes/_templates --mirror https://git.popcorntime.io/mirrors/nodecast-js.git /Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-nodecast-js-git-102c612c: Cloning into bare repository '/Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-nodecast-js-git-102c612c'...
npm ERR! git clone --template=/Users/Chase/.npm/_git-remotes/_templates --mirror https://git.popcorntime.io/mirrors/nodecast-js.git /Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-nodecast-js-git-102c612c: fatal: unable to access 'https://git.popcorntime.io/mirrors/nodecast-js.git/': Failed to connect to git.popcorntime.io port 443: Connection refused
npm ERR! git clone --template=/Users/Chase/.npm/_git-remotes/_templates --mirror https://git.popcorntime.io/mirrors/node-airplay-js.git /Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-node-airplay-js-git-2771d188: Cloning into bare repository '/Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-node-airplay-js-git-2771d188'...
npm ERR! git clone --template=/Users/Chase/.npm/_git-remotes/_templates --mirror https://git.popcorntime.io/mirrors/node-airplay-js.git /Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-node-airplay-js-git-2771d188: fatal: unable to access 'https://git.popcorntime.io/mirrors/node-airplay-js.git/': Failed to connect to git.popcorntime.io port 443: Connection refused
npm ERR! git clone --template=/Users/Chase/.npm/_git-remotes/_templates --mirror https://git.popcorntime.io/mirrors/peerflix.git /Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-peerflix-git-ba2786e4: Cloning into bare repository '/Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-peerflix-git-ba2786e4'...
npm ERR! git clone --template=/Users/Chase/.npm/_git-remotes/_templates --mirror https://git.popcorntime.io/mirrors/peerflix.git /Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-peerflix-git-ba2786e4: fatal: unable to access 'https://git.popcorntime.io/mirrors/peerflix.git/': Failed to connect to git.popcorntime.io port 443: Connection refused
npm ERR! git clone --template=/Users/Chase/.npm/_git-remotes/_templates --mirror https://git.popcorntime.io/mirrors/chromecast-js.git /Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-chromecast-js-git-a4757beb: Cloning into bare repository '/Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-chromecast-js-git-a4757beb'...
npm ERR! git clone --template=/Users/Chase/.npm/_git-remotes/_templates --mirror https://git.popcorntime.io/mirrors/chromecast-js.git /Users/Chase/.npm/_git-remotes/https-git-popcorntime-io-mirrors-chromecast-js-git-a4757beb: fatal: unable to access 'https://git.popcorntime.io/mirrors/chromecast-js.git/': Failed to connect to git.popcorntime.io port 443: Connection refused
```

**Comment by EdgarVaguencia** [comment]
+1

**Comment by hadim** [comment]
Same here.

**Comment by zerske** [comment]
It looks like the DNS records are still still live for git.popcorntime.io (A record pointed to 62.210.123.48) but the content host is down. If one of the active devs (@xaiki ? ) has access to these files they should just need to be moved to a different host, and update the faulty URLs to point to the new host.

**Comment by hfossli** [comment]
Same here

**Comment by matiastucci** [comment]
+1

**Comment by RyanBeckett** [comment]
+1

**Comment by nusson** [comment]
yep

**Comment by TheAtomicGoose** [comment]
+1

**Comment by maxwerr** [comment]
this should be resolved by https://github.com/butterproject/butter/commit/7e7b8ab615ee7eb5ad8b4cdc26ef2732a39b9ba1#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R33

**Comment by hadim** [comment]
It should work now. Try again.

**Comment by xaiki** [comment]
fixed by @hadim thanks a lot to him !

**Comment by rclai** [comment]
It's back?

`fatal: unable to access 'https://git.popcorntime.io/mirrors/chromecast-js.git/': Could not resolve host: git.popcorntime.io`

**Comment by zerske** [comment]
You might need to pull (or rebase) from remote to get the latest version of the source repository. Check out the changes made to the package.json file in the aforementioned fix:

https://github.com/butterproject/butter/issues

TL;DR it's still fixed
